chef/cookbooks/neutron/templates/default/metadata_agent.ini.erb

```diff
@@ -7,7 +7,10 @@ debug = <%= @debug ? "True" : "False" %>
 
 # The Neutron user information for accessing the Neutron API.
 # auth_url = http://localhost:5000/v2.0
-auth_url = <%= @keystone_settings['internal_auth_url'] %>
+auth_url = <%= KeystoneHelper.versioned_service_URL(@keystone_settings["protocol"],
+                                                    @keystone_settings["internal_url_host"],
+                                                    @keystone_settings["service_port"],
+                                                    "2.0") %>
```

Comment:
Will we track these locations using v2.0 so we can fix them later on? Or we just don't care and will leave them that way forever?